README.md

```diff
@@ -198,9 +198,16 @@ optional `prepare.yml` and `verify.yml` playbooks. Running the tests then
 proceeds as with testing the roles:
 
 ```shell
-molecule --base-config ../molecule_configs/centos7_base_config.yml test --scenario centos7_monitoring
+molecule test --scenario centos7_monitoring
```

Comment:
we don't use the base configs for testing playbooks (as the base configs only spin up a single instance but we need multiple for the playbooks)